web/app/components/base/button/index.tsx

```diff
@@ -1,50 +1,44 @@
-import type { FC, MouseEventHandler } from 'react'
 import React from 'react'
+import { type VariantProps, cva } from 'class-variance-authority'
+import classNames from 'classnames'
 import Spinner from '../spinner'
 
-export type IButtonProps = {
-  type?: string
-  className?: string
-  disabled?: boolean
-  loading?: boolean
-  tabIndex?: number
-  children: React.ReactNode
-  onClick?: MouseEventHandler<HTMLDivElement>
-}
+const buttonVariants = cva(
+  'btn disabled:pointer-events-none',
+  {
+    variants: {
+      variant: {
+        primary: 'btn-primary disabled:btn-primary-disabled',
+        warning:
+          'btn-warning disabled:btn-warning-disabled',
+        default: 'btn-default disabled:btn-default-disabled',
+      },
+    },
+    defaultVariants: {
+      variant: 'primary',
```

Comment:
```suggestion
      variant: 'default',
```
default variant should be 'default'

Reply:
ths!